runtime/proto/rpc_common.proto

```diff
@@ -49,13 +49,7 @@ message LayoutMsg {
  *       no longer present and that was subsumed by a checkpoint).
  */
 message StreamAddressSpaceMsg {
-  // Holds the last trimmed address for this stream.
-  // Note: keeping the last trimmed address is required in order to properly set the stream tail on sequencer resets
-  // when a stream has been checkpointed and trimmed and there are no further updates to this stream.
-  int64 trim_mark = 1;
-
-  // Holds the complete map of addresses for this stream.
-  // Note: currently of type Roaring64NavigableMap.
+  // Holds the last trimmed address for this stream and complete map of addresses for this stream.
```

Comment:
why do we remove the trim mark? we wouldn't be covering the case of a fully checkpointed stream whose last address is no longer present, but we need this to set the stream tail... I'm thinking we probably don't have a test that covers this scenario which we have seen in production..

Reply:
It's just removed from the proto model, but it is still serialized/deserialized

In `StreamAddressSpace`
```java
    public void serialize(DataOutput out) throws IOException {
        out.writeLong(trimMark);
        bitmap.serialize(out);
    }

    public static StreamAddressSpace deserialize(DataInputStream in) throws IOException {
        long trimMark = in.readLong();
        Roaring64NavigableMap map = new Roaring64NavigableMap();
        map.deserialize(in);
        return new StreamAddressSpace(trimMark, map);
    }
```

The reason I removed it is because the bitmap and the trimmark shouldn't be accessed outside of StreamAddressSpace object.